spotbugs/src/main/java/edu/umd/cs/findbugs/visitclass/DismantleBytecode.java

```diff
@@ -742,6 +742,10 @@ public void visit(Code obj) {
 
                 }
                 switch (opcode) {
+                case Const.CONSTANT_Dynamic:
+                    registerOperand = opcode - Const.CONSTANT_Dynamic;
```

Comment:
This `registerOperand` is always `0` but according to [JEP309](https://openjdk.java.net/jeps/309) it could be another value (and static analysis has no way to know it?).
I guess we need to find another approach to tackle this issue. 🤔